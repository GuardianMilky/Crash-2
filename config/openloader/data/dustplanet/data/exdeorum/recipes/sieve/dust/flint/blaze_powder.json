{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:dust"},"mesh":"exdeorum:flint_mesh","result":"minecraft:blaze_powder","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.04}}