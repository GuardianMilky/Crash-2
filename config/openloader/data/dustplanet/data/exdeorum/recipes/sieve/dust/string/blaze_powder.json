{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:dust"},"mesh":"exdeorum:string_mesh","result":"minecraft:blaze_powder","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.03}}