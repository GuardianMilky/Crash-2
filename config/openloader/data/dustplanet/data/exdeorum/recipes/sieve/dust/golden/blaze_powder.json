{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:dust"},"mesh":"exdeorum:golden_mesh","result":"minecraft:blaze_powder","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.06}}